example/lib/main.dart

```diff
@@ -31,47 +25,44 @@ import 'package:flutter_map_example/pages/reset_tile_layer.dart';
 import 'package:flutter_map_example/pages/secondary_tap.dart';
 import 'package:flutter_map_example/pages/sliding_map.dart';
 import 'package:flutter_map_example/pages/stateful_markers.dart';
-import 'package:flutter_map_example/pages/tap_to_add.dart';
 import 'package:flutter_map_example/pages/tile_builder_example.dart';
 import 'package:flutter_map_example/pages/tile_loading_error_handle.dart';
-import 'package:flutter_map_example/pages/widgets.dart';
 import 'package:flutter_map_example/pages/wms_tile_layer.dart';
+import 'package:url_strategy/url_strategy.dart';
 
-void main() => runApp(const MyApp());
+void main() {
+  setPathUrlStrategy();
```

Comment:
Can't we use `usePathUrlStrategy` instead of relying on the url_strategy dependency? Here's the related [Flutter documentation](https://docs.flutter.dev/ui/navigation/url-strategies#configuring-the-url-strategy)

Reply:
I used the dependency as it no-ops for non-web platforms. Either way, a dependency is needed (either on this or flutter_web_plugins (which I will admit is from SDK, but still needs to be installed)), and this is just less setup. If you think it's a bigger deal, I'm happy to change it.

Reply:
Well, it's not that bad, merely a suggestion (hence why I've still approved the PR), I'm okay if it stays like this for now 👍 
But as the package hasn't been updated in 2 years it might be wise to separate from it in the future before it becomes an issue.

Reply:
Sure. Although, I can't see it being changed soon - it's implementation is incredibly simple and literally boils down to calling from flutter_web_plugins and a platform dependent import.

Thanks for reviewing!